subworkflows/local/germline_variant_calling.nf

```diff
@@ -2,96 +2,92 @@
 // GERMLINE VARIANT CALLING
 //
 
-params.haplotypecaller_options        = [:]
-params.deepvariant_options            = [:]
-params.genotypegvcf_options           = [:]
-params.concat_gvcf_options            = [:]
-params.concat_haplotypecaller_options = [:]
-params.strelka_options                = [:]
-
-include { GATK_JOINT_GERMLINE_VARIANT_CALLING      } from '../../subworkflows/nf-core/joint_germline_variant_calling/main'
-include { GATK4_HAPLOTYPECALLER as HAPLOTYPECALLER } from '../../modules/nf-core/modules/gatk4/haplotypecaller/main'
-include { DEEPVARIANT }                              from '../../modules/local/deepvariant/main'
-include { GATK4_GENOTYPEGVCFS as GENOTYPEGVCF }      from '../../modules/nf-core/modules/gatk4/genotypegvcfs/main'
-include { CONCAT_VCF as CONCAT_GVCF }                from '../../modules/local/concat_vcf/main'
-include { CONCAT_VCF as CONCAT_HAPLOTYPECALLER }     from '../../modules/local/concat_vcf/main'
-include { STRELKA_GERMLINE as STRELKA }              from '../../modules/nf-core/modules/strelka/germline/main'
+include { GATK_JOINT_GERMLINE_VARIANT_CALLING } from '../../subworkflows/nf-core/joint_germline_variant_calling/main'
```

Comment:
That does look interesting.
Where is this script?

Reply:
It's from this PR nf-core/modules#1128 I have start to include it unmerged to see what still needs adapting. Typically there is multiple rounds of upstream updates I found. and I wanted to speed up the process